books_scraper/books_scraper/spiders/books.py

```diff
@@ -0,0 +1,25 @@
+import scrapy
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    start_urls = ["https://books.toscrape.com/"]
+
+    def parse(self, response):
+        for book_link in response.css("article.product_pod h3 a::attr(href)").getall():
+            yield response.follow(book_link, self.parse_book)
+
+        next_page = response.css("li.next a::attr(href)").get()
+        if next_page:
+            yield response.follow(next_page, self.parse)
+
+    def parse_book(self, response):
+        yield {
+            "title": response.css("h1::text").get(),
+            "price": response.css("p.price_color::text").get(),
+            "amount_in_stock": response.css("p.instock.availability::text").re_first("\d+"),
+            "rating": response.css("p.star-rating::attr(class)").re_first("star-rating (\w+)"),
+            "category": response.css("ul.breadcrumb li:nth-child(3) a::text").get(),
+            "description": response.css("meta[name='description']::attr(content)").get().strip(),
```

Comment:
The `strip()` method is called on the result of `get()`, which can be `None` if the description meta tag is not found. This will raise an `AttributeError`. Consider using a conditional expression to handle the `None` case, e.g., `response.css("meta[name='description']::attr(content)").get(default='').strip()`.